eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -244,32 +244,6 @@ jobs:
       # Will run samples described on a file name [service]-samples.txt within the build directory.
       # For example keyvault-samples.txt.
       # The file is written by CMake during configuration when building samples.
-    - bash: |
-        IFS=$'\n'
-        if [[ -f "./${{ parameters.ServiceDirectory }}-samples.txt" ]]; then
-          for sample in `cat ./${{ parameters.ServiceDirectory }}-samples.txt`
-          do
-            export AZURE_CLIENT_ID=$(${{parameters.ServiceDirectory}}_CLIENT_ID)
-            export AZURE_TENANT_ID=$(${{parameters.ServiceDirectory}}_TENANT_ID)
```

Comment:
What's motivating this change to move this as an else?

Reply:
the if statement is IF use AzrurePipelineCredentials define these ENVs , ti was upside down

Reply:
@weshaggard since Daniel is OOF, can you please help review the changes in this `live.tests.yml` file. I think it could benefit from an extra set of eyes to make sure it is as expected :)

Reply:
it was very easy to check , before my change the build the live tests were failing, after it was working 
and if you read the line right above it you see the if statement

Reply:
It looks correct assuming you want to continue to run the samples via default credentials instead of with pipeline credentials.

Reply:
Daniel was aware of the change, i talked to him before making it

Reply:
[7/3 10:01 AM] George Arama
from what i see in the live.tests.yml the 
- ${{ if parameters.UseFederatedAuth }}:
.....
-${{ else }}:
..
$env:AZURESUBSCRIPTION_CLIENT_ID = $account.Id

$env:AZURESUBSCRIPTION_TENANT_ID = $account.Tenants
 
...
 
the variables get defined when not using federated auth on the False branch 
 
[7/3 10:24 AM] George Arama
in the live tests yaml the if/else branches for the run  samples are flipped , i'm testing my fix for it 
 
[7/3 11:13 AM] George Arama
looks like ti's working now as expected

Reply:
resolving this conversation to merge the CI